osgp/protocol-adapter-iec60870/osgp-protocol-adapter-iec60870/src/main/java/org/opensmartgridplatform/adapter/protocol/iec60870/infra/messaging/BaseResponseEventListener.java

```diff
@@ -53,10 +57,19 @@ public void connectionClosed(final IOException e) {
     protected void sendResponse(final Serializable response) {
 
         final DeviceMessageMetadata deviceMessageMetadata = new DeviceMessageMetadata(this.messageMetadata);
-        final ProtocolResponseMessage protocolResponseMessage = new ProtocolResponseMessage.Builder()
+
+        final ProtocolResponseMessage.Builder protocolResponseMessageBuilder = new ProtocolResponseMessage.Builder()
                 .domain(this.messageMetadata.getDomain()).domainVersion(this.messageMetadata.getDomainVersion())
                 .deviceMessageMetadata(deviceMessageMetadata).result(ResponseMessageResultType.OK)
-                .retryCount(this.messageMetadata.getRetryCount()).dataObject(response).build();
+                .retryCount(this.messageMetadata.getRetryCount()).dataObject(response);
+
+        LOGGER.info("correlationIdProviderService: {}", this.correlationIdProviderService);
```

Comment:
What is the point in logging this field? Remove it.

Reply:
I've removed it.

Background: at some point in time, I had an issue with correlationIdProviderService being null. The logging statement was added to check the value. I should have removed it, when I fixed my dependency injection issue.